cocotb_test/simulator.py

```diff
@@ -210,7 +210,7 @@ def set_env(self):
 
         self.env["PYTHONHOME"] = get_config_var("prefix")
 
-        self.env["TOPLEVEL"] = self.toplevel
+        self.env["TOPLEVEL"] = self.toplevel[0]   # What is the purpose of this? Does multi toplevel break it?
```

Comment:
```suggestion
        self.env["TOPLEVEL"] = self.toplevel_module
```
This is available for the user. Not sure about the use case.

Reply:
Ok, I would think it is better to not break existing behaviour then.